### PR TITLE
Added Marathi locale #1014

### PR DESCRIPTION
**Pull request by Muskan7Tripathi**
# Description

Added Marathi Locale. By google translate, I translated the string.

Fixes (#1014 )

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

[values-mr.zip](https://github.com/Swati4star/Images-to-PDF/files/7105609/values-mr.zip)
mr
mr

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java [hunk 1]
```diff
@@ -192,7 +192,7 @@ public void openExcelToPdf() {
 
     @OnClick(R.id.open_pdf)
     void openPdf() {
-        mFileUtils.openFile(mPath);
+        mFileUtils.openFile(mPath, FileUtils.FileType.e_PDF);
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java:195]: why these changes?

**Review by Swati4star — CHANGES_REQUESTED** [review]

File: app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java (hunk 1 above)
Comment (Muskan7Tripathi) [reply app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java:195]: For listing files and directories filters.

**Review by Muskan7Tripathi — COMMENTED** [review]
For listing file and directories filters.

File: app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java (hunk 1 above)
Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java:195]: This PR is supposed to add marathi locale only, right? Then, these changes should not be a part of this PR.

**Comment by Muskan7Tripathi** [comment]
Yes ma'am . You're right

**Comment by Swati4star** [comment]
Closing due to inactivity.
